
------------ LOG | 12/06/2022 - 14:25:31 ------------
[14:25:31] - INFO - monitor 1 (8820) OK!
[14:25:31] - INFO - monitor 2 (13652) OK!
[14:25:31] - WARNING - monitor 1 (Traceback (most recent call last):
  File "c:\Users\henri\Documents\GitHub\S2SLauncher\frontend\iOneLauncher.py", line 184, in monitor_manager
    self.module_logger.write_file(["WARNING", f"Invalid URL running on monitor: ({name}) - ({driver.current_url}), corrections: ({i})"])
  File "C:\Users\henri\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 920, in current_url
    return self.execute(Command.GET_CURRENT_URL)['value']
  File "C:\Users\henri\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 430, in execute
    self.error_handler.check_response(response)
  File "C:\Users\henri\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 247, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=102.0.5005.115)
Stacktrace:
Backtrace:
	Ordinal0 [0x00ACD953+2414931]
	Ordinal0 [0x00A5F5E1+1963489]
	Ordinal0 [0x0094C6B8+837304]
	Ordinal0 [0x00934120+737568]
	Ordinal0 [0x00996E29+1142313]
	Ordinal0 [0x009A3FB2+1195954]
	Ordinal0 [0x009942B6+1131190]
	Ordinal0 [0x0096E860+976992]
	Ordinal0 [0x0096F756+980822]
	GetHandleVerifier [0x00D3CC62+2510274]
	GetHandleVerifier [0x00D2F760+2455744]
	GetHandleVerifier [0x00B5EABA+551962]
	GetHandleVerifier [0x00B5D916+547446]
	Ordinal0 [0x00A65F3B+1990459]
	Ordinal0 [0x00A6A898+2009240]
	Ordinal0 [0x00A6A985+2009477]
	Ordinal0 [0x00A73AD1+2046673]
	BaseThreadInitThunk [0x768AFA29+25]
	RtlGetAppContainerNamedObjectPath [0x778C7A7E+286]
	RtlGetAppContainerNamedObjectPath [0x778C7A4E+238]

)
[14:25:31] - WARNING - monitor 2 (Traceback (most recent call last):
  File "c:\Users\henri\Documents\GitHub\S2SLauncher\frontend\iOneLauncher.py", line 181, in monitor_manager
    if driver.current_url == blocked or not driver.current_url == application_url:
  File "C:\Users\henri\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 920, in current_url
    return self.execute(Command.GET_CURRENT_URL)['value']
  File "C:\Users\henri\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 430, in execute
    self.error_handler.check_response(response)
  File "C:\Users\henri\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 247, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=102.0.5005.115)
Stacktrace:
Backtrace:
	Ordinal0 [0x00ACD953+2414931]
	Ordinal0 [0x00A5F5E1+1963489]
	Ordinal0 [0x0094C6B8+837304]
	Ordinal0 [0x00934120+737568]
	Ordinal0 [0x00996E29+1142313]
	Ordinal0 [0x009A3FB2+1195954]
	Ordinal0 [0x009942B6+1131190]
	Ordinal0 [0x0096E860+976992]
	Ordinal0 [0x0096F756+980822]
	GetHandleVerifier [0x00D3CC62+2510274]
	GetHandleVerifier [0x00D2F760+2455744]
	GetHandleVerifier [0x00B5EABA+551962]
	GetHandleVerifier [0x00B5D916+547446]
	Ordinal0 [0x00A65F3B+1990459]
	Ordinal0 [0x00A6A898+2009240]
	Ordinal0 [0x00A6A985+2009477]
	Ordinal0 [0x00A73AD1+2046673]
	BaseThreadInitThunk [0x768AFA29+25]
	RtlGetAppContainerNamedObjectPath [0x778C7A7E+286]
	RtlGetAppContainerNamedObjectPath [0x778C7A4E+238]

)
